src/utils/languageMap.ts

```diff
@@ -0,0 +1,742 @@
+import { Language } from '../flowTypes';
+
+export const languageMap: { [key: string]: Language } = {
```

Comment:
would like to avoid having this exhaustive list here, looks like this is being used for the base language only being an iso code. we can look into options for this later tho.

Reply:
Same, yeah this is just a first pass at getting the name of a language from its iso since that's all the `set_contact_language` action and `environment` response give us RN.